core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -231,11 +248,8 @@ fun dateFormatted(publishDate: Instant): String {
         }
     }
 
-    return DateTimeFormatter
-        .ofLocalizedDate(FormatStyle.MEDIUM)
-        .withLocale(Locale.getDefault())
-        .withZone(zoneId)
-        .format(publishDate.toJavaInstant())
+    return DateTimeFormatter.ofPattern("MMM d, yyyy")
+        .withZone(zoneId).format(publishDate.toJavaInstant())
```

Comment:
This is probably an unintended change. @qamarelsafadi can you confirm?

Reply:
yes you are right, don't know what happened I will revert it now